ocd_backend/sources/osi.ibabs.yaml

```diff
@@ -10,28 +10,28 @@ _entity_defaults: &entity_defaults
   id: "{index_name}_{entity}"
   loaders:
     - ocd_backend.loaders.elasticsearch.elasticsearch_upsert_loader
-    - ocd_backend.loaders.delta.delta_loader
+    # - ocd_backend.loaders.delta.delta_loader
   cleanup: ocd_backend.tasks.cleanup_elasticsearch
 
-_motions_and_amendments_voting_defaults: &motions_and_amendments_voting_defaults
-  <<: *entity_defaults
-  extractor: ocd_backend.extractors.ibabs.IBabsReportsExtractor
-  transformer: ocd_backend.transformers.ibabs_motion.IBabsMotionItem
-  loader: ocd_backend.loaders.ElasticsearchUpsertLoader
-  include: moties
-  pdf_max_pages: 20
-  max_pages: 1
-  per_page: 100
-
-_voting_rounds_defaults: &voting_rounds_defaults
-  <<: *entity_defaults
-  entity: voting_rounds
-  doc_type: vote_events
-  extractor: ocd_backend.extractors.ibabs.IBabsVotesMeetingsExtractor
-  transformer: ocd_backend.transformers.voting_round.IBabsVotingRoundItem
-  loader: ocd_backend.loaders.ElasticsearchUpdateOnlyLoader
-  reverse_chronological: false
-  max_processed_meetings: 0
+#_motions_and_amendments_voting_defaults: &motions_and_amendments_voting_defaults
```

Comment:
Are these parts commented out permanently or temporary? If permanently all those can be removed.

Reply:
I'm not sure if they worked in the past or if they were just placeholders.